docs/root/start/sandboxes/tls-inspector.rst

```diff
@@ -0,0 +1,100 @@
+.. _install_sandboxes_tls_inspector:
+
+TLS Inspector Listener Filter
+=============================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+	Used to make ``HTTP`` requests.
+
+   :ref:`jq <start_sandboxes_setup_jq>`
+	Parse ``json`` output from the upstream echo servers.
+
+This example demonstrates how the ``TLS``  inspector can be used to select ``FilterChains`` to
+distribute the traffic between upstream clusters according to the matched ``transport_protocol`` and/or
+``application_protocols``.
+
+It also demonstrates the admin statistics generated by the ``TLS`` inspector listener filter.
+
+Step 1: Build the sandbox
+*************************
+
+Change directory to ``examples/tls-inspector`` in the Envoy repository.
+
+This starts one proxy listening on ``localhost:10000``.
+
+It also starts three upstream services, one ``HTTP1.1`` and one ``HTTP2`` over ``TLS`` and another
+pure ``HTTP``, which echo back received headers in ``json`` format.
+
+The upstream services listen on the internal Docker network on ports ``443`` and ``80`` respectively.
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/tls-inspector
+  $ docker-compose pull
+  $ docker-compose up --build -d
+  $ docker-compose ps
+
+                Name                               Command               State                         Ports
+  ---------------------------------------------------------------------------------------------------------------------------------
+  tls-inspector_service-http_1            docker-entrypoint.sh node  ... Up
+  tls-inspector_service-https-http1.1_1   docker-entrypoint.sh node  ... Up
+  tls-inspector_service-https-http2_1     docker-entrypoint.sh node  ... Up
+  tls-inspector_tls-inspector_1           /docker-entrypoint.sh /usr ... Up      0.0.0.0:10000->10000/tcp, 0.0.0.0:12345->12345/tcp
+
+
+Step 2: Access services
+***********************
+
```

Comment:
i think this needs to be broken into 3 sub-sections:

```
Query the proxy with ``HTTP1.1`` and ``TLS``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Query the proxy with ``HTTP2`` and ``TLS``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Query the proxy with no ``TLS``
~~~~~~~~~~~~~~~~~~~~~~~~

```

(underlines are supposed to be the length of title - markdown is fooing it)